source/server/core_server.cpp

```diff
@@ -88,6 +88,15 @@ static void RunServer(const ServerConfiguration& config)
         config.config_file_path.c_str());
   }
 
+  std::string string_kNiDeviceGrpcBranchName(nidevice_grpc::kNiDeviceGrpcBranchName);
```

Comment:
This is the same logic as in the --version case. Please create a static function in this file. that is then called in both places so the logic for both of these will always stay consistent. 
https://github.com/ni/grpc-device/blob/d2f2862128e30d758aff738db6964d6c6dcf36d6/source/server/core_server.cpp#L206